src/components/SearchBox.vue

```diff
@@ -21,15 +21,44 @@
         :class-names="classNames"
         v-model="currentRefinement"
       >
+        <template
+          v-slot:loading-indicator
+          v-if="isVue3"
+        >
+          <slot
+            name="loading-indicator"
+          />
+        </template>
         <slot
+          v-if="isVue2"
           name="loading-indicator"
           slot="loading-indicator"
         />
```

Comment:
what is this for? I don't understand it. Could we support only the vue-3 style with vue 2.6 support?

Reply:
Forgot to mention. 

`<slot>` alone didn't enable passing the slot from the parent down to `<search-input>` in vue 3. And wrapping it with `template` didn't work for vue 2.6 either. So I had to keep the both versions.

Reply:
is there a vue issue or documentation for this? I'm surprised it wasn't needed for the other slots then?

Reply:
I couldn't find anything related. And in case of Pagination, it's different from this.

What's particular about this is, for example,

There is a slot named `loading-indicator` in SearchBox. If a user passes something to that slot, SearchBox relays it to SearchInput (which accepts a slot with the same name). If nothing's passed to `loading-indicator`, then SearchInput fills with its default template.

I guess there's something to do with passing slots between components. On contrary, there's nothing like that in case of Pagination.

Reply:
ok let's go with this then :)